sdk/easm/Azure.Analytics.Defender.Easm/samples/README.md

```diff
@@ -0,0 +1,14 @@
+---
+page_type: sample
+languages:
+- csharp
+products:
+# Including relevant stubs from https://review.docs.microsoft.com/help/contribute/metadata-taxonomies#product
+- azure
```

Comment:
seems like there should be another of these for a more specific match.   Is `defender` a thing in MS Docs?